packages/core/typescript/itk-wasm/cypress/e2e/pipeline/run-pipeline.cy.ts

```diff
@@ -60,6 +60,20 @@ describe('runPipeline', () => {
     })
   })
 
+  it('uses a web worker created explicitly beforehand', () => {
+    cy.window().then(async (win) => {
+      const itk = win.itk
+      const pipelineWorkerUrl = new URL('/itk-wasm-pipeline.worker.js', demoServer).href
+      const webWorker = await itk.createWebWorker(pipelineWorkerUrl)
+
+      const args = []
+      const outputs = null
+      const inputs = null
+      const stdoutStderrPath = 'stdout-stderr-test'
+      const { returnValue, stdout, stderr } = await itk.runPipeline(webWorker, stdoutStderrPath, args, outputs, inputs, { pipelineWorkerUrl })
```

Comment:
Is `{ pipelineWorkerUrl }` option still needed?

Reply:
Nope! Good catch.